src/js/state/createGlobalStore.ts

```diff
@@ -1,9 +1,10 @@
 import {configureStore} from "@reduxjs/toolkit"
-import globalReducer, {GlobalState} from "./globalReducer"
+import rootReducer from "./rootReducer"
+import {State} from "./types"
 
-export default function (initState: GlobalState | undefined) {
+export default function (initState: Partial<State> | undefined) {
   return configureStore({
-    reducer: globalReducer,
+    reducer: rootReducer,
```

Comment:
I realized that this doesn't need to be a different reducer. It's just the same reducer but we only use some of it in the global store.